e2e/features/Udacity.feature

```diff
@@ -14,20 +14,21 @@ Scenario: 1. The user should be able to use the search bar
    Then the course count should be less than the remembered course count
 
    When the search bar is cleared
-   Then the course count should be equal to the remembered course count
+   Then the course count should equal to the remembered course count
 
 Scenario: 2. The card details and the filter fields should work correctly
   Given the Udacity course page is opened
     
    When the text "javascript" is typed into the search bar
-   Then the selected filters field should contain the "javascript" text
-    And the course count should be equal to the number of the courses that visible
+   Then the selected filters field should contain the filter "javascript"
+    And The course count should equal to the result counter
```

Comment:
Could you pls change the 'The' to 'the' here?
Sorry, if it comes from my comment.

Reply:
Corrected.